pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -207,7 +207,7 @@ public CompletableFuture<NamespaceEphemeralData> tryAcquiringOwnership(Namespace
      *
      */
     public CompletableFuture<Void> removeOwnership(NamespaceBundle bundle) {
-        ResourceLock<NamespaceEphemeralData> lock = locallyAcquiredLocks.get(bundle);
+        ResourceLock<NamespaceEphemeralData> lock = locallyAcquiredLocks.remove(bundle);
```

Comment:
Could you please help add a unit test? It will help to avoid any regression for this part.

Reply:
done